website/docs/CustomCommands.md

```diff
@@ -97,6 +97,48 @@ Be careful to not overload the `browser` scope with too many custom commands.
 
 We recommend defining custom logic in [page objects](pageobjects), so they are bound to a specific page.
 
+### Multiremote
+
+`addCommand` works in a similar way for multiremote, except the new command will propagate down to the children instances. You have to be mindful when using `this` object since the multiremote `browser` and its children instances have different `this`.
+
+This example shows how to add a new command for multiremote.
+
+```js
+browser.addCommand('getUrlAndTitle', async function (customVar) {
+    // `this` refers to:
+    //      - MultiRemoteBrowser scope for browser
+    //      - Browser scope for instances
+    return {
+        url: await this.getUrl(),
+        title: await this.getTitle(),
+        customVar: customVar
+    }
+})
+
+browser.getUrlAndTitle()
+/*
+{
+    url: [ 'https://webdriver.io/', 'https://webdriver.io/' ],
+    title: [
+        'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO',
+        'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO'
+    ],
+    customVar: undefined
+}
+*/
+
+browserA.getUrlAndTitle()
+/*
+{
+    url: 'https://webdriver.io/',
+    title: 'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO',
+    customVar: undefined
+}
+*/
```

Comment:
Maybe we can make this more explicit by using TypeScript:

```ts
import { multiremotebrowser } from '@wdio/globals'

multiremotebrowser.addCommand('getUrlAndTitle', async function (this: WebdriverIO.MultiRemoteBrowser, customVar: any) {
    // `this` refers to:
    //      - MultiRemoteBrowser scope for browser
    //      - Browser scope for instances
    return {
        url: await this.getUrl(),
        title: await this.getTitle(),
        customVar: customVar
    }
})

multiremotebrowser.getUrlAndTitle()
/*
{
    url: [ 'https://webdriver.io/', 'https://webdriver.io/' ],
    title: [
        'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO',
        'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO'
    ],
    customVar: undefined
}
*/

multiremotebrowser.getInstance('browserA').getUrlAndTitle()
/*
{
    url: 'https://webdriver.io/',
    title: 'WebdriverIO · Next-gen browser and mobile automation test framework for Node.js | WebdriverIO',
    customVar: undefined
}
*/
```

Reply:
better?